### PR TITLE
Fix notification circle (border-radius)

### PR DESCRIPTION
**Pull request by wxiaoguang**
`border-radius` means `radius`, not `diameter`, so it should be `50%` and `boxHeight / 2`

image

### CODE REVIEW

**Comment by puni9869** [comment]
What will be single digit notification looks like. could you share a screenshot for that.

**Review by puni9869 — APPROVED** [review]

**Comment by wxiaoguang** [comment]
> What will be single digit notification looks like. could you share a screenshot for that.

The same as before, no change.

image

File: web_src/css/modules/navbar.css [hunk 1]
```diff
@@ -133,8 +133,8 @@
   left: 6px;
   top: -9px;
   min-width: 17px;
-  min-height: 17px;
-  border-radius: var(--border-radius-circle);
+  height: 17px;
+  border-radius: 11px; /* (height + 2 * borderThickness) / 2 */
```
Comment (silverwind) [inline web_src/css/modules/navbar.css:137]: Not sure about the comment but value is right, matching `height`.

**Review by silverwind — APPROVED** [review]

File: web_src/css/modules/navbar.css (hunk 1 above)
Comment (wxiaoguang) [reply web_src/css/modules/navbar.css:137]: It (the comment) is right, just rounds up.

Actually, `border-radius:` could be any large enough value, like 1000px

**Comment by puni9869** [comment]
🚀  let's roll.

File: web_src/css/modules/navbar.css (hunk 1 above)
Comment (silverwind) [reply web_src/css/modules/navbar.css:137]: Ah I missed that you changed value as well. Well let's hope it works :)

**Pull request merged**